### ISSUE TITLE
Radar 42

### ISSUE THREAD

**Issue by whit537** [gratipay/inside.gratipay.com#472]
What are you working on this week and why?

[last week](https://github.com/gratipay/inside.gratipay.com/issues/462)

**Comment by whit537** [comment]
#### Roadmap ([what?](http://inside.gratipay.com/howto/sweep-the-radar))

###### Short Term

[Embarrassments](https://github.com/gratipay/inside.gratipay.com/issues/368#issuecomment-148587948):

 - can't edit teams ([Team Basics](https://github.com/gratipay/gratipay.com/milestones/Team%20Basics))
 - not using immutable ids (https://github.com/gratipay/gratipay.com/issues/835)
 - widgets are broken (https://github.com/gratipay/inside.gratipay.com/issues/389#issuecomment-153884921)
 - escrow imprecision (https://github.com/gratipay/inside.gratipay.com/issues/389#issuecomment-153929881)
 - no payroll! https://github.com/gratipay/gratipay.com/issues/3433
 - Team and ~user pages are the same https://github.com/gratipay/gratipay.com/issues/3852

###### Long Term

 Color code:

 - red—external forces that we're under
 - yellow—administration
 - green—product development
 - orange—marketing
 - blue—capitalization


[![roadmap](https://cloud.githubusercontent.com/assets/134455/11450312/7c0546ac-9569-11e5-957c-4c9898666661.png)](http://interactive.blockdiag.com/?compression=deflate&src=eJyNUsFu2zAMvfsrBO3aXQsURjckXlsE6NAAHnpZdmBkOiUqS4akOEuL_PtE2bGDpWjrk_hE6vG957W26rki2IjXLBMbgFb81rBGfS3vZrOlvBDKauuuHVYXIuDf0Je7Jwr4R2SiVTQOLIuF-FGWH85gM5HcNK22-wZNEPewe380E2sPExukMf-ZOaXHscI2LRjCT8wJdsTZbStee1h-ub0t4ifz5FTO6nOWk_NmeaQ5ZL0lX78JMrX1qHq5sW5h76zWg8kRkHPQYBSKubXPXvbqIj77eZ82jsc726Ez3BTfHR4clSxiXaKSUR9V3QRX0RUK-5V5REc1KQhkDXcNG5w6yPVogtw4RCNPjZDJiURhOkuKzOaEf0D4Oi49XsTzQGe3MZ8jXJFDFVZmwLmlACj_S2YvwK8MiBJdR4qpz0Ko46dUNOzcwsNZc1FwZtx89I-ziX6liAYFfNtDY0pHgIX1OC894bx6rKaETkD5C6ERc_CkvGR8SZ0NfHgwawuuSpTnu7Ksy0vedWEC68edT3_K0qH33_niYUvlEzh8a_zqSqm6Tr4U0FIATS99-Nx9-Ae_HEZp)

**Comment by whit537** [comment]
Inbox 5, GitHub 0, L2 Support 0, Vendors, etc. 0.

**Comment by whit537** [comment]
Pretty skimpy info (re: https://github.com/gratipay/inside.gratipay.com/issues/462#issuecomment-171524090). Sending the book back.

![the big bro](https://cloud.githubusercontent.com/assets/134455/12419958/50e6a064-be87-11e5-9043-1f5c9f0bb824.jpg)

![picture credits](https://cloud.githubusercontent.com/assets/134455/12420876/2f18e46e-be8d-11e5-88c3-fcbef15c7944.jpg)

**Comment by whit537** [comment]
Here's another project using Aspen!

https://github.com/falstaff84/acdr

:-)

**Comment by whit537** [comment]
> In turn, banks are starting to properly deal with payment aggregators, which were previously ignored if you were too small, protected if you were generating enough revenue, or shut down if you were in the awkward middle ground. Back then aggregators weren’t required to collect KYC and AML info because they were largely unregulated. They should have been collecting it, but no one was making them do it.

http://jkwade.me/post/128191359172/underwriting-marketplace-merchants

Sounds about right. :disappointed_relieved:

**Comment by whit537** [comment]
Teams reviewed.

**Comment by mattbk** [comment]
We're increasing the number of teams, but ~users who give is remaining pretty flat. What can we do to improve that?
- Gain new ~users
- Encourage old ~users who were receiving to convert to teams
- ?

**Comment by mattbk** [comment]
On a related note, https://gratipay.com/~visionmedia is a prime example of a "team of one" who was getting lots of support under Gratipay 1.0.

**Comment by whit537** [comment]
Alright! Looks like @rohitpaulk just brough his first complete payday in for a landing (#477). Woo-hoo! !m @rohitpaulk. :airplane: Meanwhile, I've been processing some awesome work from @jaraco on [Aspen](https://github.com/gratipay/aspen.py/pulls?utf8=%E2%9C%93&q=is%3Apr+author%3Ajaraco+created%3A2016-01-19..2016-01-21) (and [Mongs](https://github.com/whit537/mongs/commits/master?author=jaraco)). !m @jaraco

My goals for the afternoon are to draft something on #457 (deadline is this weekend), and to get escrow to show up on the income statement over on https://github.com/gratipay/finances/pull/1. @kaguillera has started on 2012-07. We are on track to have one or both months landed ahead of meeting with the CPA in a week (#285).

**Comment by whit537** [comment]
It's gonna be huge to have an accounting system in place. We'll be able to approach our next growth curve with *much* more confidence, knowing *exactly* where we stand. We'll have a clear way to add new payin and payout routes such as [Dwolla](https://github.com/gratipay/gratipay.com/issues/65) and [Transpay](https://github.com/gratipay/gratipay.com/issues/417) without feeling like we're getting in over our heads ... same for potential modifications such as [monthly](https://github.com/gratipay/gratipay.com/issues/3862) and [one-off](https://github.com/gratipay/gratipay.com/issues/5) payments. It ain't flashy, but proper accounting is vital to Gratipay's success. :chart_with_upwards_trend:

**Comment by rohitpaulk** [comment]
> It ain't flashy, but proper accounting is vital to Gratipay's success.

:+1:

**Comment by whit537** [comment]
> We're increasing the number of teams, but ~users who give is remaining pretty flat. What can we do to improve that?

The basic answer is the same as always: a) improve the product and b) tell people about it. I think you're right that the [fuzzy ~user/Team distinction](https://github.com/gratipay/gratipay.com/issues/3852) probably slows a lot of people down, for example.

**Comment by mattbk** [comment]
Would adding a ~~call to action~~ special note on ~users without teams pages help (perhaps once we make the pages distinct)?

> "This user is not on any teams, but if they're doing something cool and you want to give to them, why not tag them on social media and ask them to start a team?"

Sort of like the old way where you could give to people before they had an account (caught the mention of that in the call on Tuesday; didn't realize it had been dropped). CTA could be something like  we have here: https://gratipay.com/on/twitter/tj/.

**Comment by whit537** [comment]
> Would adding a call to action special note on ~users without teams pages help

Sounds good to me.

**Comment by whit537** [comment]
I'm cutting a release and I'm not sure I got the `1937` tag from the remote as I would've expected when updating. Not sure what's going on and I'm back on track now, chalking it up to "something to keep an eye out for in the future."

**Comment by whit537** [comment]
https://github.com/gratipay/finances/pull/1 is ready to merge! Check out the docs here:

https://github.com/gratipay/finances/tree/4514deab5d4926aa9e5f1e620033dc99e8d148ab#readme

:dancer:

**Comment by kzisme** [comment]
Awesome work everyone :dart:

**Comment by whit537** [comment]
Thanks @kzisme! :-)

**Comment by techtonik** [comment]
> We're increasing the number of teams, but ~users who give is remaining pretty flat. What can we do to improve that?

The biggest incentive for me to use Gratipay was to redistribute the $$$ I earned to other open source projects. Now that I don't have anything incoming, it is kind of useless to me.

Also, for the past week I did a Roundup release and earned 500 EUR. To receive the money I had to create invoice, or make an agreement for the scope of work, send bank details and receive payment.  But I can't do this twice a year, because I don't have legal entity. I can receive money only as a gift, which is what original Gratipay is proposed. So I am still interested - can I still receive gifts from people sent by Gratipay LLC.? On my side it is somewhat clear that I can, but what about the sender?
